include/mixer.h

```diff
@@ -24,10 +24,17 @@
 #include "dosbox.h"
 #endif
 
+#include <functional>
+
 #include "envelope.h"
 
 typedef void (*MIXER_MixHandler)(Bit8u *sampdate, Bit32u len);
-typedef void (*MIXER_Handler)(Bitu len);
+
+// The mixer callback can accept a static function or a member function
+// using a std::bind. The callback typically requests enough frames to fill
+// one millisecond. For an audio channel running at 48,000 Hz, that's 48
+// frames.
```

Comment:
Nitpick, because the meaning is clear based on the rest of comment, but lease update the comment to say 48 kHz. Motivation: https://en.wikipedia.org/wiki/Decimal_separator

Basically, from POV of e.g. polish or french speaker, 48,000 means 48 while 48 000 or 48 kHz won't lead to any misunderstandings :)

Reply:
Oh, excellent point.  Yes - will fix this up.